### ISSUE TITLE
Monitor not working in multiplayer

### ISSUE THREAD

**Issue by simcolin** [cc-tweaked/cc-restitched#47]
### Minecraft Version

1.18.x

### Version

1.99.1

### Details

There is a problem with monitors in multiplayer, they seems to not work at all, I've tested the same code to write on the monitor in solo and it worked, but in multiplayer the screen stay black the whole time.

### Test code :
```lua
local mon = peripheral.find("monitor")
mon.setCursorPos(1, 1)
mon.write("Test")
```

### Logs : 
Absolutely no logs or crashed about that problem, but I will link the logs since the last serveur reboot where I've tested the code above : [latest.log](https://github.com/cc-tweaked/cc-restitched/files/7783993/latest.log)

### Detailed reproduction step :
I suppose you just have to start a multiplayer fabric server with the version of CC I'm talking about. Although the problem might come from an incompatibility with some of the other mods I installed on the serveur (see latest.log for full mod list server side).

### Bonus info that might help :
On the client we have a few more mods that are not on the server. I tried poping out Iris because I saw on your README that compatibility with it was not yet perfect, but it didn't affect anything.
Here is the list of mods other than iris that with have client side :

- 3D Skin Layer
- Entity Culling
- Roughly Enough Item (with his 2 dependency mods)
- Mod Menu
- Not Enought Animations

**Comment by simcolin** [comment]
### Update
There are some crashes related to CC and monitors when saving the world [2021-12-28-2.log](https://github.com/cc-tweaked/cc-restitched/files/7784459/2021-12-28-2.log)

**Comment by simcolin** [comment]
### Update 2

The bug seems to be fixed with the lastest edge build for 1.18 [here](https://github.com/cc-tweaked/cc-restitched/actions/runs/1625069115)
